scripts/date-logic/date-logic.js

```diff
@@ -54,5 +54,30 @@
     { '$count': 'numFormFields' }
   ]
   db.getCollection('forms').aggregate(afterScriptFormFieldsWithIsFutureOnlyTrue)
+ 
 
-  
\ No newline at end of file
+// REMOVE ISFUTUREONLY KEY 
+
+  // Number of forms with isFutureOnly key - expect 0 after running update
+
+  db.forms.count({ 'form_fields': { '$elemMatch': { 'isFutureOnly': {$exists: true}  } } })
+
+  // Number of form fields with isFutureOnly key - expect 0 after running update
+
+  const formFieldsWithIsFutureOnlyExist = [
+    { '$match': { 'form_fields': { '$elemMatch': { 'isFutureOnly': {$exists: true} } } } },
+    { '$project': { 'form_fields': 1 } },
+    { '$unwind': '$form_fields' },
+    { '$match': { 'form_fields.isFutureOnly': {$exists: true} } },
+    { '$count': 'numFormFields' }
+  ]
+  
+  db.getCollection('forms').aggregate(formFieldsWithIsFutureOnlyExist)
+
+  // Update - Remove isFutureOnly key
+
+  db.forms.update(
+    {"form_fields.isFutureOnly": {$exists: true}}, 
+    {"$unset": {"form_fields.$[elem].isFutureOnly": ""}}, 
```

Comment:
Not sure if this will unset the form field? I thought you'd have to specify `1`. Not that sure, @mantariksh?

```suggestion
    {"$unset": {"form_fields.$[elem].isFutureOnly": 1}}, 
```

Reply:
Shouldn't need:

> The specified value in the $unset expression (i.e. "") does not impact the operation.
https://docs.mongodb.com/manual/reference/operator/update/unset/